include/mbedtls/aes.h

```diff
@@ -121,14 +121,14 @@ typedef struct mbedtls_aes_xts_context
  *                 It must be the first API called before using
  *                 the context.
  *
- * \param ctx      The AES context to initialize.
+ * \param ctx      The AES context to initialize. Must not be \c NULL.
```

Comment:
These additions should be turned into sentences.

Reply:
Thanks, I will fix it.

Reply:
Oh, that's actually part of the base PR.